doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -714,7 +714,12 @@ nRF RPC libraries
 Other libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`lib_date_time` library:
+
+  * Fixed a bug that caused date-time updates to not be rescheduled under certain circumstances.
+  * Added a retry feature that reattempts failed date-time updates up to a certain number of consecutive times.
+  * Added the :kconfig:option:`DATE_TIME_RETRY_COUNT` Kconfig option to control whether and how many consecutive date-time update retries may be performed.
+  * Added the :kconfig:option:`DATE_TIME_RETRY_INTERVAL_SECONDS` Kconfig option to control how quickly date-time update retries occur.
```

Comment:
`CONFIG_` prefix needed to get the link generated properly in the documentation.

```suggestion
  * Added the :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT` Kconfig option to control whether and how many consecutive date-time update retries may be performed.
  * Added the :kconfig:option:`CONFIG_DATE_TIME_RETRY_INTERVAL_SECONDS` Kconfig option to control how quickly date-time update retries occur.
```

Reply:
And the "added" entries should be moved under a common "Added" heading.